packages/CollabCodeEdit-Core.package/CCEEdit.class/README.md

```diff
@@ -0,0 +1,11 @@
+I am an edit transferred to or from another one by the CCE-Suite. I hold all the information necessary to apply myself. I am an abstract class.
+
+Instance Variables
+	classIdentifier:	Symbol
+	editContent:		Object
+
+classIdentifier
+	- The classIdentifier is the name of the class edited.
```

Comment:
Minor quibble: If the title is "classIdentifiert" and the first two words are "the classIdentifier is the" I do not get anything of value. Please rephrase, perhaps to something like "The name of the edited class".